packages/clayui.com/src/pages/docs/getting_started/introduction.md

```diff
@@ -2,30 +2,246 @@
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules)/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        compact: false,
+                        presets: ['env']
+                    }
+                }
+            },
+            {
+                test: /\.scss$/,
+                use: [
+                    {
+                        loader: 'file-loader',
+                        options: {
+                            name: 'bundle.css',
+                        },
+                    },
+                    {
+                        loader: 'extract-loader',
+                    },
+                    {
+                        loader: 'css-loader',
+                    },
+                    {
+                        loader: 'sass-loader',
+                    }
+                ]
+            }
+        ]
+    }
+};
+```
+
+To test the webpack configuration, run:
+
+```shell
+npm run start
+```
+
+And it opens `http://localhost:8080` in the browser. You should see `Hello World!` in red color.
+
+### Step 2: Install Clay component
+
+In order to install Clay you can choose to install the separate packages or install with all packages, see [Importing the JS Component](/docs/getting_started/importing-the-js-component.html). To proceed with this we will install only the `ClayBadge` package, consider running the command to do this:
+
+```shell
+npm install --save clay-badge
+```
+
+Next, do the import of `ClayBadge` in `app.js`:
+
+```javascript
+import ClayBadge from 'clay-badge';
+import './app.scss';
+```
+
+If you look at the screen you will see the same thing as before, but we will go forward to know how to use the `ClayBadge`.
+
+### Step 3: Install Clay css
+
+To install `clay-css`, consider running the command:
+
+```shell
+npm install --save clay-css
+```
+
+In order for `sass-loader` to see Clay's `.scss` files, configure `webpack.config.js` the loader `sass-loader` options. You need to import the `clay-css` and path package from node.js to move on:
+
+```javascript
+const clay = require('clay-css');
+const path = require('path');
+```
+
+Configure the loader `sass-loader` in `webpack.config.js` to see the clay:
+
+```javascript
+// ...
+{
+    loader: 'sass-loader',
+    options: {
+        includePaths: clay
+            .includePaths
+            .concat(
+                path.join(
+                    clay.includePaths[0],
+                    '../fonts'
+                )
+            )
+    }
+}
+// ...
 ```
 
+Then do the `clay-css` import in your `app.scss`:
+
+```text/scss
+@import "atlas";
+```
+
+See the screen in the browser and you can already see some changes of `clay-css` in action.
+
+### Step 4: Using component of Clay
+
+Since we already have our component imported into our JS, we can start using it, for this we need to create an instance of our component in `app.js`:
```

Comment:
Update:

> Since we already have our component imported into our JS, we can start using it. To do this, we must create an instance of our component in `app.js`: